### PR TITLE
Move resources to events-workshops

### PR DESCRIPTION
**Pull request by praneetloke**
You should be able to review each of these in the individual commits:

* Added the alias for `/resources` in `events-workshops/_index.md`
* Updated references to the marketing `/resources` (not the `/docs/intros/concepts/resources`) to `/events-workshops`
* Updated the marketing nav template (`header.html` partial) to point to `/events-workshops` instead of `/resources`

### CODE REVIEW

**Comment by susanev** [comment]
it was intended to make this change when we released the learn platform as defined in https://github.com/pulumi/pulumi-hugo/issues/617 though i don't feel super strongly about holding. @infin8x do you have opinions?

**Comment by praneetloke** [comment]
@susanev yeah I misunderstood the timing of the change. I'll defer to you all on whether you want to combine this PR with something else.

**Comment by infin8x** [comment]
Ah, yeah, I'd expected to see this roll out alongside the Learn Pulumi launch. But, I don't see any harm in rolling it out now - means that the Learn Pulumi gets a little simpler since we only need to add a new item, not also make these changes. If @susanev and @praneetloke are good with these changes going out now, I am too 🚀.

**Comment by praneetloke** [comment]
@infin8x I am good with it. To be clear, I worked on this with a different frame of mind and wasn't aware of the Learn Pulumi stuff. I was just breaking down another piece of work and thought this would be good to do separately than club with another thing I am working on.

**Comment by infin8x** [comment]
Sounds good - let's ship it if @zchase is 👍🏻

**Review by zchase — COMMENTED** [review]
Do the resource content files need `aliases`? We have links out in the wild that point to these pages and I don't think I saw anything in the PR that would correctly redirect those links to right place.

**Comment by praneetloke** [comment]
> Do the resource content files need `aliases`? We have links out in the wild that point to these pages and I don't think I saw anything in the PR that would correctly redirect those links to right place.

Hmm I hadn't thought of that. I came across two that already had aliases for typos in the title but yeah the content would need aliases with the `/resources` prefix. I'll update the PR.

**Comment by praneetloke** [comment]
@zchase could you please have a look at the commit e775c92851db7e644652e2d646326e56ef8831c5 for the redirects and let me know if it looks good? I also added a silly little tool to help with adding aliases for content in bulk.

**Comment by praneetloke** [comment]
Ping

**Review by susanev — COMMENTED** [review]
looks good to me, but id like Zack to review before merging

one thing i noticed is that the all filter when you navigate to events and resources is not highlighted so maybe the link should be to `events-workshops/#all` .. this is my fault since i said events-workshops without the anchor, this is my fault

File: themes/default/content/events-workshops/pulumi-update-2020-04-29/index.md
Comment (praneetloke) [inline themes/default/content/events-workshops/pulumi-update-2020-04-29/index.md:15]: I changed the indentation here to match what we are using for lists. Calling this out to not miss the addition of the alias `/resources/pulumi-update-2020-04-29` as the first item here.

**Comment by praneetloke** [comment]
@susanev 

> one thing i noticed is that the all filter when you navigate to events and workshops is not highlighted so maybe the link should be to `events-workshops/#all` .. this is my fault since i said events-workshops without the anchor, sry about that

Good catch! I have updated the links in the `header.html` partial in the latest commit to reflect that.

**Review by zchase — APPROVED** [review]
LGTM

**Pull request merged**
